modules/installation-configuration-parameters.adoc

```diff
@@ -359,7 +359,7 @@ ifdef::openshift-origin[]
 Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OVNKubernetes`.
 endif::openshift-origin[]
 ifndef::openshift-origin[]
-Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OpenShiftSDN`.
+Either `OpenShiftSDN` or `OVNKubernetes`. `OpenShiftSDN` is the provider for all-Linux networks. `OVNKubernetes` is the provider for hybrid networks that contain both Linux and Windows servers. The default value is `OpenShiftSDN`.
```

Comment:
In 4.12 default plugin is OVNKubernetes.

Reply:
This should be ok as this change goes in 4.11 and lower. How about `OpenShiftSDN is the provider for all-Linux networks. OVNKubernetes is the provider for linux networks as well as for hybrid networks that contain both Linux and Windows servers` ?